content/docs/integrations/confluence/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Confluence Search
+publishedDate: '2022-06-30T10:00:00.0Z'
+description: How to enable Confluence indexing for Roadie Search
+
+humanName: Confluence
+logoImage: '../../../assets/logos/confluence/confluence-icon.svg'
+integrationType: Integration
+---
+
+## Introduction
+
+Roadie supports integration with Atlassian Confluence wiki platform to enable cross application search functionality. Roadie indexes documents from Confluence so they are available for you to search within the Roadie search bar. 
+
+
+## Prerequisites
+
+You'll need an Atlassian account with an ability to create an API token and knowledge of the name of your Confluence organization.
+
+## Configuring Confluence Integration
+
+### Generate Atlassian API token
+
+Roadie uses Atlassian Confluence API to retrieve indexable information from the wiki platform. To be able to connect to your Confluence instance, we need an API token. 
+You can create an Atlassian token by navigating to your [**Atlassian** profile security page](https://id.atlassian.com/manage-profile/security) and Clicking 'Create and Manage API Tokens'.
+![Creating an Atlassian token](./confluence_api_token_generation.png)
+
+
+On the page navigated into you are able to generate API tokens which can be used to connect Roadie to your Confluence workspaces.
+
+### Store Token as a Secret to Roadie
+
+When you have successfully generated your token, you can navigate to `https://<your-roadie-instance>.roadie.so/administration/settings/secrets` and populate the secret value `CONFLUENCE_TOKEN`.
+
+![Confluence Secret Token Screen](./confluence_secret.png)
+
+### Configure Confluence Search Indexing
+
+Final step to enable the Confluence wiki for search indexing is to configure your Confluence organization name, username tied to the token and the cadence you want your Confluence documents to be indexed. You can navigate to `https://<your-roadie-instance>.roadie.so/administration/settings/confluence` to input the needed values. The minimum interval is 15 minutes to prevent excessive load on the used API token and Confluence API. 
```

Comment:
```suggestion
To enable your Confluence wiki for search indexing,  navigate to `https://<your-roadie-instance>.roadie.so/administration/settings/confluence` and input your organization name, username, Confluence API token, and indexing interval. The fastest interval rate we support is scheduling indexes every 15 minutes to prevent excessive use of Confluence's API token.
```

"Minimum of 15 minutes" is a bit confusing IMO (also in the UI). The meaning of the question is a bit ambiguous.